packages/grafana-data/src/field/fieldOverrides.ts

```diff
@@ -95,23 +97,22 @@ export function applyFieldOverrides(options: ApplyFieldOverrideOptions): DataFra
   }
 
   return options.data.map((frame, index) => {
-    let name = frame.name;
-    if (!name) {
-      name = `Series[${index}]`;
-    }
-
     const scopedVars: ScopedVars = {
-      __series: { text: 'Series', value: { name } },
+      __series: { text: 'Series', value: { name: getFrameDisplayTitle(frame, index) } }, // might be missing
```

Comment:
this is a problem as this makes __series.name to equal labels of the first field, and the default getTitleTemplate generates titles like `${__series.name}${__field.labels}`  which means field title with duplicated labels